frameworks/shared/callee.py

```diff
@@ -86,3 +89,12 @@ def load_data(name, path, **_):
         kill_proc_tree(include_parent=False, timeout=5)
 
     json_dump(res, config.result_file, style='compact')
+
+def measure_inference_times(predict_fn: Callable[[str], Any], files: list[Tuple[int, str]]) -> dict[int, list[float]]:
+    inference_times = defaultdict(list)
+    for subsample_size, subsample_path in files:
+        for _ in range(10):
```

Comment:
Just a heads up: for `arcene` and `Mercedes_Benz_Greener_Manufacturing` you might run into very long run-times for this call when predicting 10,000 rows with 10 repeats depending on the framework.

The current logic of AMLB is to terminate if the process goes on for too long, so you might get some failures due to this on otherwise successful frameworks. This can be solved by disabling the termination logic or reducing the number of repeats.

Reply:
Thanks for the heads up (again). I am not sure of a nice way to make this work automatically. On some level, it's still useful to have automatic termination (some frameworks won't stop otherwise), so I imagine we best increase it across the board. Perhaps I will rerun only those specific datasets with even more lenient (or absent) termination times.